src/app/GitExtUtils/GitUI/Interops/COMBOBOXINFO.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System.Runtime.InteropServices;
+
+namespace System;
+
+internal static partial class NativeMethods
+{
+    [StructLayout(LayoutKind.Sequential)]
+    internal struct COMBOBOXINFO
+    {
+        public int cbSize;
+        public RECT rcItem;
+        public RECT rcButton;
+        public ComboBoxButtonState buttonState;
+        public IntPtr hwndCombo;
+        public IntPtr hwndEdit;
+        public IntPtr hwndList;
+
+        public static COMBOBOXINFO Create() => new() { cbSize = Marshal.SizeOf<COMBOBOXINFO>() };
```

Comment:
This is not needed.

Also, `Marshal.*` calls should be avoided due to perf overhead.

Reply:
It says here next to `cbSize` https://learn.microsoft.com/en-us/windows/win32/api/winuser/ns-winuser-comboboxinfo
```
The size, in bytes, of the structure. The calling application must set this to sizeof(COMBOBOXINFO).
```

I've changed this to be a one-time calculation when setting a `static readonly` field. Or what did you have in mind?

Reply:
No, structs can't have any properties or fields as that changes the size and layout of the struct.
Methods are, generally, acceptable but there's no compelling reason here.